lib/getPostcssResult.js

```diff
@@ -104,7 +104,15 @@ module.exports = function(
       const source = options.code ? options.codeFilename : options.filePath;
       let preProcessedCode = code;
 
-      if (options.codeProcessors) {
+      if (options.codeProcessors && options.codeProcessors.length) {
+        if (stylelint._options.fix) {
+          // eslint-disable-next-line no-console
+          console.warn(
+            "`--fix` cmd flag is not compatible with `processors` in configuration, so it will be disabled."
```

Comment:
Shall we say?:

```suggestion
            "Autofix is incompatible with processors and will be disabled. Are you sure you need a processor?"
```

As `fix` can also be enabled via the Node API. The "Are you sure..." bit is because I suspect a lot of users might be using processors needlessly for things like `<style>` tags.